docs/features/autoupdater.rst

```diff
@@ -11,14 +11,19 @@ during development), but it can be enabled by setting the variable GLUON_BRANCH
 to override the default branch set in the set in the site configuration.
 
 A manifest file for the updater can be generated with `make manifest`. A signing script (using
-ecdsautils) can by found in the `contrib` directory. When creating the manifest, ``GLUON_PRIORITY`` can
-be set on the command line, or it can be taken from the ``site.mk``.
+``ecdsautils``) can by found in the `contrib` directory. When creating the manifest, PRIORITY can
+be set on the command line, or it can be taken from the ``site.mk``, by the variable ``GLUON_PRIORITY``.
```

Comment:
but it **is** called `PRIORITY`, see the [autoupdater page](https://github.com/freifunk-gluon/gluon/blob/v2016.2.1/docs/releases/v2014.3.rst#new-autoupdater):

>PRIORITY    Specifies the maximum number of days until the update should be attempted (thus lower numbers mean the priority is higher). It must be set either in site.mk or on the make manifest command line.